tools/apprunner-gradle-plugin/build.gradle

```diff
@@ -4,7 +4,12 @@ plugins {
   id 'com.gradle.plugin-publish' version '0.12.0'
 }
 
-version = file('version.txt').text.trim()
+def quarkusVersion = file('../../pom.xml').readLines().find {l -> l.contains('<quarkus.version>') }.replaceFirst(~/.*<quarkus.version>(.*)<\/quarkus.version>.*/, '$1' )
+if (!file('version.txt').exists()) {
+  version = file('../pom.xml').readLines().find {l -> l.contains('<version>') }.replaceFirst(~/.*<version>(.*)<\/version>.*/, '$1' )
```

Comment:
regex for parsing xml make me sad :(

Reply:
TIL there's `mvn help:evaluate` - added a commit to this PR